src/test/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/controller/EventControllerIntTest.kt

```diff
@@ -41,6 +50,18 @@ class EventControllerIntTest : IntegrationTestBase() {
     courtCasesQueue?.sqsClient?.purgeQueue(PurgeQueueRequest.builder().queueUrl(courtCasesQueue!!.queueUrl).build())
     val str = File("src/test/resources/json/court-application-minimal.json").readText(Charsets.UTF_8)
     hearingEvent = objectMapper.readValue(str, HearingEvent::class.java)
+    amazonS3.createBucket(CreateBucketRequest.builder().bucket(bucketName).build())
+    amazonS3.createBucket(CreateBucketRequest.builder().bucket(largeCasesBucketName).build())
+  }
+
+  @AfterEach
+  fun afterEach() {
+    val objectListing = amazonS3.listObjects(ListObjectsRequest.builder().bucket(bucketName).build()).get()
```

Comment:
Is this necessary?
In practice there will be other items in the bucket, which is short-lived anyway.